requirements.txt

```diff
@@ -1,16 +1,15 @@
+AfricastalkingGateway==1.9
 coverage==4.4.2
 dj-database-url==0.4.2
 django-appconf==1.0.1
 django-ses-backend==0.1.1
 Django==1.10
 django_compressor==2.1
 djmail==0.11.0
+freezegun==0.3.9
```

Comment:
what does `freezegun` do?

Reply:
That came around when Flavia was writing tests for the assigned task.

Reply:
it's being used in the test file `hc/api/tests/test_sendreports.py` to  mock the default 
 datetime module using `freeze_time`, needed it for our test file to run and to be sure.